.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -73,7 +73,7 @@ This PR affects the following parts of the plugin, which may require extra testi
 
 ## Documentation
 
-* [ ] I have written documentation for this change.
+* [ ] I have written documentation for this change. For example, comments in the Relevant technical choices, comments in the code, documentation on Confluence / shared Google Drive / [Yoast developer portal](https://developer.yoast.com/), or other.
```

Comment:
I think here we should also ask all information provided in the issue ticket. For example in the ticket I am working on, I gathered relevant documentation from the issue itself.